tests/basic/node_decommission_test.go

```diff
@@ -181,7 +181,7 @@ var _ = Describe("{DecommissionNode}", func() {
 
 					return false, true, fmt.Errorf("node %s not joined yet", nodeToDecommission.Name)
 				}
-				_, err = task.DoRetryWithTimeout(t, appReadinessTimeout, defaultRetryInterval)
+				_, err = task.DoRetryWithTimeout(t, 20*time.Minute, defaultRetryInterval)
```

Comment:
Any particular reason we did this change?

Reply:
I have noticed sometimes after labelling the nodes to restart especially kvdb nodes, there is a race condition between operator induced restart and px-daemon restart leading to take some time to px to be up